File: 4-go/models/payment.go
package models

import (
	"gorm.io/gorm"
)

type Payment struct {
	gorm.Model
	ID          uint    `gorm:"primaryKey" json:"id"`
	IdOrder     uint    `gorm:"column:id_order;not null" json:"idOrder"`
	PaymentType uint    `gorm:"column:payment_type;not null" json:"paymentType"`
	Amount      float64 `gorm:"column:amount" json:"amount"`
	PaymentDate string  `gorm:"column:payment_date;not null" json:"paymentDate"`
	Status      uint    `gorm:"column:status" json:"status"`
}

func (Payment) TableName() string {
	return "payment"
}

File: 5-react/my-app/src/models/payment.js
export class Payment {
    id;
    id_order;
    payment_type;
    amount;
    payment_date;
    status;

    constructor(id, id_order, payment_type, amount, payment_date, status) {
        this.id = id;
        this.id_order = id_order;
        this.payment_type = payment_type;
        this.amount = amount;
        this.payment_date = payment_date;
        this.status = status;
    }
}